Sources/SwiftGen/Config/ConfigEntry.swift

```diff
@@ -15,6 +15,7 @@ import PathKit
 struct ConfigEntry {
   enum Keys {
     static let inputs = "inputs"
+    static let filter = "filter"
     static let outputs = "outputs"
 
     // Legacy: remove this once we stop supporting the output key at subcommand level
```

Comment:
Unrelated to this PR, but where are we on versioning, will the next version be a minor or a major one? Could be the occasion with the new CoreData support and since the deprecation has been present for a while now to cleanup those…?

Reply:
6.1.0, but it's up to us when to remove deprecated stuff. I'd say remove them in one of the next minor versions, or the next major (which might be sooner than later).